src/main/java/pl/anicos/nmp/publish/model/NpmPublishBody.java

```diff
@@ -22,9 +23,6 @@
     @Valid
     private final Map<String, Attachment> attachments;
 
-    @NotEmpty
-    @Valid
-
     @JsonCreator
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) "distTags" is marked "javax.validation.constraints.NotNull" but is not initialized in this constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2637)